src/__screenshot_tests__/success-feedback-screen-with-navbar-screenshot-test.tsx

```diff
@@ -0,0 +1,25 @@
+import {openStoryPage} from '../test-utils';
+import {MOVISTAR_SKIN, O2_NEW_SKIN} from '../skins/constants';
+
+import type {Device} from '../test-utils';
+
+const testableSkins = [MOVISTAR_SKIN, O2_NEW_SKIN];
+const testableDevices: Array<Device> = ['MOBILE_IOS', 'DESKTOP'];
+
+const cases: Array<[string, Device]> = [];
+for (const skin of testableSkins) {
+    for (const device of testableDevices) {
+        cases.push([skin, device]);
+    }
+}
+
+test.each(cases)('SuccessFeedbackScreen with NavigationBar - %s - %s', async (skin, device) => {
+    const page = await openStoryPage({
+        id: `private-fixed-footer--default`,
+        skin: skin as never,
```

Comment:
I'd follow the same thing we have in all the other tests in mistica, where we declare the skin array (in this case, `testableSkins`) as const:

`const testableSkins = [MOVISTAR_SKIN, O2_NEW_SKIN] as const;`

And then do something like this:

```suggestion
        skin: skin as (typeof testableSkins)[number],
```

Reply:
ok, done